### ISSUE TITLE
Wiki Incomplete??

### ISSUE THREAD

**Issue by xXxToxicBlueDustxXx** [Sygmei/ObEngine#150]
Someone Link trow me at localhost

**Comment by Sygmei** [comment]
Yup the wiki is incomplete for now, feel free to join the Discord if you have questions :)

**Comment by Sygmei** [comment]
Related to these issues :
- https://github.com/Sygmei/ObEngine/issues/68
- https://github.com/Sygmei/ObEngine/issues/70
- https://github.com/Sygmei/ObEngine/issues/71
- https://github.com/Sygmei/ObEngine/issues/72
- https://github.com/Sygmei/ObEngine/issues/73
- https://github.com/Sygmei/ObEngine/issues/74

**Comment by Sygmei** [comment]
Closing issue as it's a duplicate
